passkey-authenticator/src/credential_store.rs

```diff
@@ -32,10 +55,14 @@ pub trait CredentialStore {
         cred: Passkey,
         user: PublicKeyCredentialUserEntity,
         rp: PublicKeyCredentialRpEntity,
+        options: Options,
```

Comment:
I was thinking that it might be nice to have if the application wants to add additional credential protections based on how the credential was created, i.e. something like `credProtect` even if `passkey-rs` doesn't yet have support for it. I can replace it with `rk: boolean` if you think it's unnecessary

Reply:
I think we can stick with options for now, the `cred-protect` like behavior is indeed important.